mmpretrain/apis/feature_extractor.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Callable, List, Optional, Union
+
+import numpy as np
+import torch
+from mmcv.image import imread
+from mmengine.config import Config
+from mmengine.dataset import Compose, default_collate
+from mmengine.device import get_device
+from mmengine.infer import BaseInferencer
+from mmengine.model import BaseModel
+from mmengine.runner import load_checkpoint
+
+from mmpretrain.registry import TRANSFORMS
+from .model import get_model, list_models
+
+ModelType = Union[BaseModel, str, Config]
+InputType = Union[str, np.ndarray, list]
+
+
+class FeatureExtractor(BaseInferencer):
+    """The inferencer for extract features.
```

Comment:
add comments to indicates why we don't implement `visualize` amd `postprocess`.

e.g   "FeatureExtractor aims to extract the features of backbone, it doesn't require  `visualize` and `postprocess`"

Reply:
No, it's just because we haven't completed it.